### PR TITLE
Version updated of Glob to 1.1.5

### LINKED ISSUE #422: Please update Glob v1.1.5 to remove debug logging
Could you please update `Glob` dependency to version 1.1.5 to remove the debug or hard coded console.writeline logging.

https://github.com/kthompson/glob/releases/tag/1.1.5
https://github.com/kthompson/glob/commit/711b3293e9923cfcd26c8ab34565573bf1181e05,

Thx

### PR DESCRIPTION
**Pull request by cmenzi**
I confirm that the pull-request:

- [x] Follows the contribution guidelines
- [x] Is based on my own work
- [x] Is in compliance with my employer

Fixes #422

### CODE REVIEW

**Comment by matkoch** [comment]
could you rebase the other commit out of the pr?

File: build/Build.AzurePipelinesAttribute.cs [hunk 1]
```diff
@@ -15,8 +15,11 @@ partial class Build
 {
     public class AzurePipelinesAttribute : Nuke.Common.CI.AzurePipelines.AzurePipelinesAttribute
     {
-        public AzurePipelinesAttribute(AzurePipelinesImage image, params AzurePipelinesImage[] images)
-            : base(image, images)
+        public AzurePipelinesAttribute(
+            string suffix,
```
Comment (ryancyq) [inline build/Build.AzurePipelinesAttribute.cs:19]: this looks like a breaking change but it was released in PATCH version?

Comment (matkoch) [reply build/Build.AzurePipelinesAttribute.cs:19]: Sorry, but until we hit 1.0 there is no guarantee for breaking changes. However, I'm trying to keep them as small as possible, and you also can always ask in our [Slack workspace](https://slofile.com/slack/nukebuildnet).

Comment (matkoch) [reply build/Build.AzurePipelinesAttribute.cs:19]: @ryancyq from next hotfix release there will be a new constructor allowing to not pass a suffix.. Sorry I forgot thinking of a second constructor to make it optional.

https://github.com/nuke-build/nuke/pull/456

Comment (ryancyq) [reply build/Build.AzurePipelinesAttribute.cs:19]: Cool, thanks for the update 👍🏻

**Pull request merged**